exporters/otlp/src/main/java/io/opentelemetry/exporters/otlp/OtlpGrpcMetricExporter.java

```diff
@@ -89,7 +90,7 @@ private OtlpGrpcMetricExporter(ManagedChannel channel, long deadlineMs) {
    * @return the result of the operation
    */
   @Override
-  public ResultCode export(Collection<MetricData> metrics) {
+  public CompletableResultCode export(Collection<MetricData> metrics) {
```

Comment:
@carlosalberto We will need to change this from `MetricsServiceBlockingStub` to `MetricsServiceFutureStub` to see the main result of this PR, making at least the gRPC exporters truly async. I'm ok with that being in a followup but do you want to see it now?

@huntc I guess it's not a huge code change so maybe can knock it out.

Reply:
That'd be great @anuraaga - could you make a PR on my branch?

Reply:
@huntc Sure sent https://github.com/huntc/opentelemetry-java/pull/1

Reply:
Merged - thanks @anuraaga. @carlosalberto You've now got a fully async exporter.